roles/pmm/tasks/alerting-alerts.yaml

```diff
@@ -1,3 +1,11 @@
+- name: Current Alert
+  ansible.builtin.debug:
+    msg: "{{ pmm_alert_item.name }}"
+
+- name: Set Alert Filter
+  ansible.builtin.set_fact:
+    pmm_alerting_filters: "{% if pmm_alert_item.name is defined and pmm_alert_item.name in pmm_alert_filters %}{{ pmm_alert_filters[pmm_alert_item.name][\"filters\"] | default([])}}{% else %}[]{% endif %}"
```

Comment:
What is this? This isn't how other conditional values are being set.

Reply:
For every alert rule it has to check the filters if the alert rule name is in there, if not it need to have a default. This is the only way how I was able to achieve that but if you have better ideas please let me know.

Reply:
You seem to be missing the point, go and look how the other conditional values are being set.. certainly not like this

Reply:
For example: 
```
roles/pmm/vars/main.yaml
77:   engine: '{{ "DISCOVER_RDS_POSTGRESQL" if group_names | intersect(["postgresql"]) else "DISCOVER_RDS_MYSQL" }}'
80:   port: '{{ 5432 if group_names | intersect(["postgresql"]) else 3306 }}'
```